doc/conf.py

```diff
@@ -236,12 +236,15 @@
 
 sphinx_gallery_conf = {
     'doc_module': 'sklearn',
+    'backreferences_dir': os.path.join('modules', 'generated'),
     'reference_url': {
         'sklearn': None,
         'matplotlib': 'http://matplotlib.org',
         'numpy': 'http://docs.scipy.org/doc/numpy-1.6.0',
         'scipy': 'http://docs.scipy.org/doc/scipy-0.11.0/reference',
-        'nibabel': 'http://nipy.org/nibabel'}
+        'nibabel': 'http://nipy.org/nibabel'},
```

Comment:
Not sure how nibabel got in there (probably from nilearn at one point) but we should remove it since nibabel is not used in scikit-learn examples.